fcrepo/fcrepo
# Actions

**JIRA Ticket**: https://jira.lyrasis.org/browse/FCREPO-3532

# What does this Pull Request do?

Replaces Travis with Github Actions

# How should this be tested?

This is difficult to test. Here is an example of the build workflow: https://github.com/pwinckles/fcrepo4/actions/runs/363679840

I have not tested the deploy workflow. I think it should work, but it may need tweaking. All I can say for certain is that it is syntactically correct.

The deploy workflow only runs on pushes to `main` and the following secrets must be defined in the Github settings:

* OSSRH_USERNAME
* OSSRH_TOKEN
* GPG_PASSPHRASE
* GPG_PRIVATE_KEY
* DOCKER_USERNAME
* DOCKER_PASSWORD

**I have not defined these secrets.**

# Interested parties
@fcrepo/committers
